var/spack/repos/builtin/packages/chill/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+# ----------------------------------------------------------------------------
+from spack import *
+
+
+class Chill(AutotoolsPackage):
+    """A polyheadral compiler for autotuning"""
+
+    homepage = "http://github.com/CtopCsUtahEdu"
+    url      = "https://github.com/CtopCsUtahEdu/chill/archive/v0.3.tar.gz"
+    git      = "https://github.com/CtopCsUtahEdu/chill.git"
+
+    version('master', branch='master')
+
+    depends_on('rose@0.9.10.0 +cxx11 ^boost@1.66.0 cxxstd=11')
+    depends_on('iegenlib')
+    depends_on('isl')
+    depends_on('python')
+
+    @run_before('configure')
+    def bootstrap(self, spec, prefix):
+        bash = wich('bash')
+        bash('./bootstrap')
```

Comment:
Is this standard for Autotools?  Why does this not look like the typical "autorebuild" stuff?

Reply:
This is a non-standard script that runs autoreconf and also generates test cases.

Reply:
OK.. then you probably need this in your package:
```
depends_on('autoconf', type='build')
depends_on('automake', type='build')
depends_on('libtool',  type='build')
depends_on('m4',       type='build')
```

https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html